### PR TITLE
Updates to address issues 42 through 50 (chapter 4)

### PR DESCRIPTION
**Pull request by rsahita**

### CODE REVIEW

**Comment by rsahita** [comment]
cc @ozkoyuncu

File: specification/images/img_0.png
Comment (ozkoyuncu) [inline specification/images/img_0.png:1]: Fix COVEH-ABI to say COVH-ABI

File: specification/images/img_1.png
Comment (ozkoyuncu) [inline specification/images/img_1.png:1]: Same comment: COVEH-ABI --> COVH-ABI
What does it mean the color annotation says "In application TEE TCB", I think this figure should get the same updates as the previous one

**Review by ozkoyuncu — CHANGES_REQUESTED** [review]
some minor changes on typos

File: specification/images/img_0.png
Comment (rsahita) [reply specification/images/img_0.png:1]: ack

File: specification/images/img_1.png
Comment (rsahita) [reply specification/images/img_1.png:1]: ack

File: specification/overview.adoc [hunk 1]
```diff
@@ -64,8 +65,12 @@ for itself or for non-confidential VMs.
 Each TVMs address space can be comprised of confidential and non-confidential
```
[the remark this answers is not in the document]
Comment (rsahita) [reply specification/overview.adoc:65]: ack

**Comment by rsahita** [comment]
@ozkoyuncu updated PR

**Review by ozkoyuncu — APPROVED** [review]

**Pull request merged**